swimlane/core/resources/report.py

```diff
@@ -59,10 +59,11 @@ class Report(APIResource, PaginatedCursor):
     )
 
     default_limit = 50
+    default_page_size = 10
 
     def __init__(self, app, raw, **kwargs):
         APIResource.__init__(self, app._swimlane, raw)
-        PaginatedCursor.__init__(self, limit=kwargs.pop('limit', self.default_limit))
+        PaginatedCursor.__init__(self, limit=kwargs.pop('limit', self.default_limit), page_size=kwargs.pop('page_size', self.default_page_size))
```

Comment:
We spoke about passing all report parameters. I do think we might have more in swimlane. Can you check?

Reply:
Looks like groupBys, aggregates, applicationIds, columns, and sorts are the other ones that would be useful (as opposed to, say, offset). Is that a good list?